rasa/nlu/components.py

```diff
@@ -649,17 +655,43 @@ def can_handle_language(cls, language: Hashable) -> bool:
             `True` if component can handle specific language, `False` otherwise.
         """
 
-        # if language_list is set to `None` it means: support all languages
+        # If both `supported_language_list` and `not_supported_language_list` are set to `None`,
+        # it means: support all languages
         if language is None or (
             cls.supported_language_list is None
             and cls.not_supported_language_list is None
         ):
             return True
 
-        language_list = cls.supported_language_list or []
-        not_supported_language_list = cls.not_supported_language_list or []
+        # check language supporting settings
+        if cls.supported_language_list and cls.not_supported_language_list:
+            # When user set both language supporting settings to not None, it will lead to ambiguity.
+            raise Exception(
```

Comment:
We introduced a convention to use `RasaException` instead of `Exception` for all exceptions we raise from inside the code. Helps to manage these better. Can you change to that please?
You can import it from `rasa.shared.exceptions`

Reply:
suggestion accepted